src/transport/xqc_conn.c

```diff
@@ -3790,10 +3790,10 @@ xqc_conn_update_flow_ctl_settings(xqc_connection_t *conn)
 }
 
 xqc_int_t
-xqc_conn_add_path_cid_sets(xqc_connection_t *conn, uint32_t start, uint32_t end)
+xqc_conn_add_path_cid_sets(xqc_connection_t *conn, uint64_t start, uint64_t end)
```

Comment:
why? path id is less than 2^32-1

Reply:
We use u64 for all path ids to avoid casting type everywhere.